docs_user/modules/openstack-backend_services_deployment.adoc

```diff
@@ -136,6 +138,11 @@ oc set data secret/osp-secret "SwiftPassword=$SWIFT_PASSWORD"
 * Deploy OpenStackControlPlane. *Make sure to only enable DNS,
 MariaDB, Memcached, and RabbitMQ services. All other services must
 be disabled.*
+
+* If the source environment does not use TLS Everywhere, modify
+spec:tls:endpoint:internal:enabled to false. Do the same for
+spec:tls:endpoint:public:enabled.
```

Comment:
Replace with:

On RHOSO 18, TLS Everywhere is enabled by default.  If the source environment does not use TLS Everywhere
and TLS Everywhere is not desired for the new environment, modify spec:tls:endpoint:internal:enabled to false.